scripts/bootstrap-rootfs-overlay-hosted-server.sh

```diff
@@ -28,6 +29,10 @@ while (("$#")); do
             output_dir="${2}"
             shift 2
             ;;
+        -r | --region)
+            region="${2}"
+            shift 2
+            ;;
         *)
             echo "Sorry but the provided option is not supported: $1"
             echo "Usage:  $(basename $0) --output-dir ./input/rootfs_overlay_demo --tenant-token <paste your token here>"
```

Comment:
Option should be documented as optional here.

Reply:
please tell me if the new formatting is ok

Reply:
Looks good!